app/code/Magento/CatalogUrlRewrite/Observer/CategoryProcessUrlRewriteMovingObserver.php

```diff
@@ -103,13 +103,19 @@ public function execute(\Magento\Framework\Event\Observer $observer)
                 ScopeInterface::SCOPE_STORE,
                 $category->getStoreId()
             );
+            $catProdRewritesEn = $this->isCategoryRewritesEnabled();
```

Comment:
Please use readable non-abreviated variable names, `cat` and `prod` are understandable, but `en` isn't. So in this case I would suggest `$catProdRewritesEnabled`.

Thanks! 🙂

Reply:
Hi @hostep. Thank you for reviewing my PR. I have added my chnages regarding the variable name. However I replaced it to $catRewritesEnabled because it contains less than 20 symbols, so phpmd passed that checking successfully.